### PR TITLE
Add io.github.quodlibet.ExFalso

### PR DESCRIPTION
**Pull request by lazka**
This is more or less a copy of https://github.com/flathub/io.github.quodlibet.QuodLibet
since they both share the same source.

This uses a commit on master instead of a stable release, but it's in freeze mode right
now and a release will follow shortly.

### CODE REVIEW

**Comment by lazka** [comment]
bot, build io.github.quodlibet.ExFalso

File: io.github.quodlibet.ExFalso.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+      - "-DBUILD_TESTS=off"
+      - "-DBUILD_TOOLS=off"
+      - "-DAUDIO_PROCESSOR_LIB=dummy"
+    sources:
+      - type: archive
+        url: https://github.com/acoustid/chromaprint/releases/download/v1.4.3/chromaprint-1.4.3.tar.gz
+        sha256: ea18608b76fb88e0203b7d3e1833fb125ce9bb61efe22c6e169a50c52c457f82
+  - name: gst-libav
```
Comment (bilelmoussaoui) [inline io.github.quodlibet.ExFalso.yaml]: Already in the runtime

File: io.github.quodlibet.ExFalso.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  - "--talk-name=com.canonical.Unity.Launcher"
+  - "--talk-name=org.enlightenment.wm.service"
+  - "--talk-name=org.kde.plasmashell"
+
+  # ...various other plugins
+  - "--talk-name=org.freedesktop.FileManager1"
+  - "--talk-name=org.gnome.SessionManager"
+  - "--talk-name=org.xfce.FileManager"
+
+  # ...talking to dconf
```
Comment (bilelmoussaoui) [inline io.github.quodlibet.ExFalso.yaml]: Dconf permissions are not needed anymore

File: io.github.quodlibet.ExFalso.yaml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      - type: archive
+        url: https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.16.2.tar.xz
+        sha256: 5500415b865e8b62775d4742cbb9f37146a50caecfc0e7a6fc0160d3c560fbca
+  - name: libgme
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://bitbucket.org/mpyne/game-music-emu/downloads/game-music-emu-0.6.2.tar.xz
+        sha256: 5046cb471d422dbe948b5f5dd4e5552aaef52a0899c4b2688e5a68a556af7342
+  - name: gst-plugins-bad
```
Comment (bilelmoussaoui) [inline io.github.quodlibet.ExFalso.yaml]: Already in the runtime

File: io.github.quodlibet.ExFalso.yaml (hunk 1 above)
Comment (lazka) [reply io.github.quodlibet.ExFalso.yaml]: The runtime version doesn't include WMA support

File: io.github.quodlibet.ExFalso.yaml (hunk 2 above)
Comment (lazka) [reply io.github.quodlibet.ExFalso.yaml]: ok, thanks, I'll have a look

File: io.github.quodlibet.ExFalso.yaml (hunk 3 above)
Comment (lazka) [reply io.github.quodlibet.ExFalso.yaml]: I need the gst plugins for chromaprint and game-music-emu

**Comment by lazka** [comment]
bot, build io.github.quodlibet.ExFalso

File: io.github.quodlibet.ExFalso.yaml (hunk 2 above)
Comment (lazka) [reply io.github.quodlibet.ExFalso.yaml]: works nicely, thanks

**Comment by nedrichards** [comment]
cool, lgtm - once that upstream release has been made we can merge and push.

**Comment by lazka** [comment]
ok, thanks!

**Comment by lazka** [comment]
bot, build io.github.quodlibet.ExFalso

**Comment by lazka** [comment]
> cool, lgtm - once that upstream release has been made we can merge and push.

updated to use the new upstream release

**Comment by lazka** [comment]
Thanks!

**Pull request closed without merging**